console-subscriber/src/init.rs

```diff
@@ -0,0 +1,83 @@
+use crate::TasksLayer;
+use std::thread;
+use tokio::runtime;
+use tracing_core::{subscriber::Subscriber, Dispatch};
+use tracing_subscriber::{fmt, layer::Layered, prelude::*, EnvFilter, Layer, Registry};
+
+type ConsoleSubscriberLayer =
+    Layered<TasksLayer, Layered<EnvFilter, Layered<fmt::Layer<Registry>, Registry>>>;
+
+/// Starts the console subscriber server on its own thread
+///
+/// This function represents the easiest way to get started using
+/// tokio-console.
+///
```

Comment:
can we also document that this calls `tracing::subscriber::set_global_default`? i think i mentioned that earlier but the comment may have disappeared when moving files around.

Reply:
Addressed this in f2b6de51565b504f83b784f6de8f891fab929578 — sorry for moving stuff around during review